testing/jormungandr-integration-tests/src/common/jormungandr/process.rs

```diff
@@ -251,9 +252,22 @@ impl Drop for JormungandrProcess {
         }
         // There's no kill like overkill
         let _ = self.child.kill();
-
         // FIXME: These should be better done in a test harness
         self.child.wait().unwrap();
+
+        if self.temp_dir.is_some() {
+            let temp_dir = self.steal_temp_dir().unwrap();
+            if panicking() {
+                println!(
+                    "persisting node temp_dir after panic: {:?}",
+                    temp_dir.path()
+                );
+                temp_dir.into_persistent();
```

Comment:
We should save the log content to a file in the dir as well

Reply:
I'm not clear on this, when the temp_dir has files inside, and the temp_dir is persisted, the files inside survive (unless there's a place where the `node.log` file is being deleted and that I'm unaware of.

Reply:
What I mean is that there is no such `node.log` file already in the dir, because we only collect the logs in memory. In pratice, depending on the exact configuration there might be cases in which there is a `node.log` file, but that is completely empty (due to limitations to the current subscribers), and will ultimately be removed by https://github.com/input-output-hk/jormungandr/pull/3145.